101-vm-with-data-management-gateway/README.md

```diff
@@ -0,0 +1,34 @@
+# Create a data management gateway and make it workable in azure VM
+
+Before deploying the template you must have the following
+
+1. **Data Factory.** The gateway is created in the data factory. If you don't have a data factory,  see the [Create data factory](https://docs.microsoft.com/en-us/azure/data-factory/data-factory-move-data-between-onprem-and-cloud#create-data-factory) for steps to create one.
+2. **Virtual Network.** The virtual machine will join this VNET. If you don't have one, use this tutorial, see [Create virtual network](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-create-vnet-arm-pportal#create-a-virtual-network) to create one.
```

Comment:
Data management gateway is designed to do copy activity from on-prem to cloud. The gateway vm should be in the same vnet with data sources. so in most cases our customers always have a vnet already which has the ability to access data sources.